build/building/env-vars.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Environment variables for Docker Build
+description: Use environment variables to configure builds
+keywords: docker build, buildx, buildkit, env, environment variables, config
+---
+
+You can set the following environment variables to enable, disable, or change
+the behavior of features related to building:
+
+| Variable                                                          | Type              |
+| ----------------------------------------------------------------- | ----------------- |
+| [BUILDX_BUILDER](#buildx_builder)                                 | String            |
+| [BUILDX_CONFIG](#buildx_config)                                   | String            |
+| [BUILDX_EXPERIMENTAL](#buildx_experimental)                       | Boolean           |
+| [BUILDX_GIT_INFO](#buildx_git_info)                               | Boolean           |
+| [BUILDX_GIT_LABELS](#buildx_git_labels)                           | String \| Boolean |
+| [BUILDX_NO_DEFAULT_ATTESTATIONS](#buildx_no_default_attestations) | Boolean           |
+| [BUILDX_NO_DEFAULT_LOAD](#buildx_no_default_load)                 | Boolean           |
+
+You can express Boolean values for environment variables in different ways. For
+example, `true`, `1`, and `T` all evaluate to true. Evaluation is done using the
+`strconv.ParseBool` function in the Go standard library. See
+[reference documentation](https://pkg.go.dev/strconv#ParseBool) for details.
+
+## BUILDX_BUILDER
+
+Overrides the configured builder instance. Same as the `docker buildx --builder`
+CLI flag.
+
+Usage:
+
+```console
+$ export BUILDX_BUILDER=my-builder
+```
+
+## BUILDX_CONFIG
+
+_Introduced in Buildx v0.5_
+
+You can use `BUILDX_CONFIG` to specify the directory containing the BuildKit
+configuration file you want to use. Docker uses the following lookup order for
+the configuration file:
+
+- `$BUILDX_CONFIG/buildkitd.default.toml`
+- `$DOCKER_CONFIG/buildx/buildkitd.default.toml`
+- `~/.docker/buildx/buildkitd.default.toml`
+
+Usage:
+
+```console
+$ BUILDX_CONFIG=/usr/local/etc docker build -t foo .
+```
+
+## BUILDX_EXPERIMENTAL
+
+Enables experimental build features.
+
+Usage:
+
+```console
+$ export BUILDX_EXPERIMENTAL=1
+```
+
+## BUILDX_GIT_INFO
+
+_Introduced in Buildx v0.10_
```

Comment:
For now I just made this italicized text but we could also create an include for it... if we wanted to make this "version callout" a standard practice.

Reply:
Could we link to the release notes? Like https://docs.docker.com/build/release-notes/#0100